src/main/java/de/doubleslash/keeptime/view/SettingsController.java

```diff
@@ -407,8 +411,8 @@ private void initImportButton() {
             Alert informationDialog = new Alert(AlertType.INFORMATION);
             informationDialog.setTitle("Import done");
             informationDialog.setHeaderText("The data was imported.");
-            informationDialog.setContentText(
-                  "KeepTime will now be CLOSED! You have to RESTART it again to see the changes");
+            informationDialog.getDialogPane().setContent(new Label("KeepTime will now be CLOSED!\n" + "You have to RESTART it again to see the changes"));
+            informationDialog.initOwner(thisStage);

```

Comment:
can you retest the behavior after we now merged #107 where we removed the "initOwner" of most of the dialogs?